Sat  4 Jan 23:15:24 AEDT 2020
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
WARNING:tensorflow:From run_cnn.py:82: DatasetV1.make_initializable_iterator (from tensorflow.python.data.ops.dataset_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `for ... in dataset:` to iterate over a dataset. If using `tf.estimator`, return the `Dataset` object directly from your input function. As a last resort, you can use `tf.compat.v1.data.make_initializable_iterator(dataset)`.
WARNING:tensorflow:From run_cnn.py:295: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:32: conv1d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.keras.layers.Conv1D` instead.
WARNING:tensorflow:From /home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/ops/init_ops.py:1251: calling VarianceScaling.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fddb8b974a8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fddb8b974a8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:42: batch_normalization (from tensorflow.python.layers.normalization) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.BatchNormalization instead.  In particular, `tf.control_dependencies(tf.GraphKeys.UPDATE_OPS)` should not be used (consult the `tf.keras.layers.batch_normalization` documentation).
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fddb8b974a8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fddb8b974a8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde87e59b0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde87e59b0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde8805c18>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde8805c18>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:76: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde87e5c50>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde87e5c50>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde86fa748>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde86fa748>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde87e5f60>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde87e5f60>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde868ef28>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde868ef28>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde8738828>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde8738828>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:79: max_pooling1d (from tensorflow.python.layers.pooling) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.MaxPooling1D instead.
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fdde87db8d0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fdde87db8d0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde659b710>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde659b710>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde659b710>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde659b710>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde862cc88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde862cc88>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde862cc88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde862cc88>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde6467dd8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde6467dd8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde6467dd8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde6467dd8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde8606828>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde8606828>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fde1fa68048>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fde1fa68048>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde87cb4a8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde87cb4a8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde87cb4a8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde87cb4a8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde632ce10>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde632ce10>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde632ce10>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde632ce10>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde62396a0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde62396a0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde62396a0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde62396a0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde868ef28>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde868ef28>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fdde8738828>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fdde8738828>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde649ef60>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde649ef60>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde649ef60>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde649ef60>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde641dcc0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde641dcc0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde641dcc0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde641dcc0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde62265c0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fdde62265c0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde62265c0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde62265c0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde87cb630>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde87cb630>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fde1fa68048>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fde1fa68048>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:97: flatten (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.flatten instead.
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7fdde884cb00>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7fdde884cb00>>: AttributeError: module 'gast' has no attribute 'Num'
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:8: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dense instead.
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fdde62c0470>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fdde62c0470>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde643e470>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde643e470>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fde1fa68048>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fde1fa68048>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde8738828>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fdde8738828>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fddb8b97438>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fddb8b97438>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-01-04 23:15:30.903840: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcuda.so.1
2020-01-04 23:15:30.912703: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties: 
name: Tesla M40 major: 5 minor: 2 memoryClockRate(GHz): 1.112
pciBusID: 0000:82:00.0
2020-01-04 23:15:30.912947: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-04 23:15:30.914725: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-04 23:15:30.916002: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-04 23:15:30.916326: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-04 23:15:30.918275: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-04 23:15:30.919546: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-04 23:15:30.923013: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-04 23:15:30.924965: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0
2020-01-04 23:15:30.925307: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-01-04 23:15:31.007774: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x5489770 executing computations on platform CUDA. Devices:
2020-01-04 23:15:31.007849: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Tesla M40, Compute Capability 5.2
2020-01-04 23:15:31.030471: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2200040000 Hz
2020-01-04 23:15:31.034356: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x6963490 executing computations on platform Host. Devices:
2020-01-04 23:15:31.034403: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
2020-01-04 23:15:31.036628: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties: 
name: Tesla M40 major: 5 minor: 2 memoryClockRate(GHz): 1.112
pciBusID: 0000:82:00.0
2020-01-04 23:15:31.036705: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-04 23:15:31.036730: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-04 23:15:31.036753: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-04 23:15:31.036777: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-04 23:15:31.036799: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-04 23:15:31.036821: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-04 23:15:31.036844: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-04 23:15:31.040965: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0
2020-01-04 23:15:31.041040: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-04 23:15:31.043900: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1181] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-01-04 23:15:31.043927: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1187]      0 
2020-01-04 23:15:31.043942: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 0:   N 
2020-01-04 23:15:31.047247: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10265 MB memory) -> physical GPU (device: 0, name: Tesla M40, pci bus id: 0000:82:00.0, compute capability: 5.2)
2020-01-04 23:15:31.349832: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
WARNING:tensorflow:From run_cnn.py:370: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Starting train loop at step 0
2020-01-04 23:15:36.198873: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-04 23:15:39.403715: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
INFO:tensorflow:Step: 10000 - Test batch complete: accr = 0.6219561688311688
INFO:tensorflow:Step: 20000 - Test batch complete: accr = 0.7165178571428571
INFO:tensorflow:Step: 30000 - Test batch complete: accr = 0.747463474025974
INFO:tensorflow:Step: 40000 - Test batch complete: accr = 0.7408685064935064
INFO:tensorflow:Step: 50000 - Test batch complete: accr = 0.7957589285714286
INFO:tensorflow:Step: 60000 - Test batch complete: accr = 0.775974025974026
INFO:tensorflow:Step: 70000 - Test batch complete: accr = 0.8196022727272727
INFO:tensorflow:Step: 80000 - Test batch complete: accr = 0.8017451298701299
INFO:tensorflow:Step: 90000 - Test batch complete: accr = 0.8049918831168831
INFO:tensorflow:Step: 100000 - Test batch complete: accr = 0.8340097402597403
INFO:tensorflow:Step: 110000 - Test batch complete: accr = 0.8241680194805194
INFO:tensorflow:Step: 120000 - Test batch complete: accr = 0.8128043831168831
INFO:tensorflow:Step: 130000 - Test batch complete: accr = 0.841213474025974
INFO:tensorflow:Step: 140000 - Test batch complete: accr = 0.8495332792207793
INFO:tensorflow:Step: 150000 - Test batch complete: accr = 0.8578530844155844
INFO:tensorflow:Step: 160000 - Test batch complete: accr = 0.8525771103896104
INFO:tensorflow:Step: 170000 - Test batch complete: accr = 0.8558238636363636
INFO:tensorflow:Step: 180000 - Test batch complete: accr = 0.8445616883116883
INFO:tensorflow:Step: 190000 - Test batch complete: accr = 0.859577922077922
INFO:tensorflow:Step: 200000 - Test batch complete: accr = 0.8599837662337663
INFO:tensorflow:Step: 210000 - Test batch complete: accr = 0.8676948051948052
INFO:tensorflow:Step: 220000 - Test batch complete: accr = 0.867086038961039
INFO:tensorflow:Step: 230000 - Test batch complete: accr = 0.8642451298701299
INFO:tensorflow:Step: 240000 - Test batch complete: accr = 0.8692167207792207
INFO:tensorflow:Step: 250000 - Test batch complete: accr = 0.8659699675324676
INFO:tensorflow:Step: 260000 - Test batch complete: accr = 0.869926948051948
INFO:tensorflow:Step: 270000 - Test batch complete: accr = 0.8718547077922078
INFO:tensorflow:Step: 280000 - Test batch complete: accr = 0.8702313311688312
INFO:tensorflow:Step: 290000 - Test batch complete: accr = 0.8666801948051948
INFO:tensorflow:Step: 300000 - Test batch complete: accr = 0.8700284090909091
INFO:tensorflow:Step: 310000 - Test batch complete: accr = 0.8691152597402597
INFO:tensorflow:Step: 320000 - Test batch complete: accr = 0.8692167207792207
INFO:tensorflow:Step: 330000 - Test batch complete: accr = 0.8745941558441559
INFO:tensorflow:Step: 340000 - Test batch complete: accr = 0.8784496753246753
INFO:tensorflow:Step: 350000 - Test batch complete: accr = 0.8739853896103896
INFO:tensorflow:Step: 360000 - Test batch complete: accr = 0.8733766233766234
INFO:tensorflow:Step: 370000 - Test batch complete: accr = 0.877739448051948
INFO:tensorflow:Step: 380000 - Test batch complete: accr = 0.8735795454545454
INFO:tensorflow:Step: 390000 - Test batch complete: accr = 0.8740868506493507
INFO:tensorflow:Step: 400000 - Test batch complete: accr = 0.875101461038961
INFO:tensorflow:Step: 410000 - Test batch complete: accr = 0.8783482142857143
INFO:tensorflow:Step: 420000 - Test batch complete: accr = 0.8808847402597403
INFO:tensorflow:Step: 430000 - Test batch complete: accr = 0.8731737012987013
INFO:tensorflow:Step: 440000 - Test batch complete: accr = 0.8771306818181818
INFO:tensorflow:Step: 450000 - Test batch complete: accr = 0.8778409090909091
INFO:tensorflow:Step: 460000 - Test batch complete: accr = 0.8709415584415584
INFO:tensorflow:Step: 470000 - Test batch complete: accr = 0.8693181818181818
INFO:tensorflow:Step: 480000 - Test batch complete: accr = 0.8746956168831169
INFO:tensorflow:Step: 490000 - Test batch complete: accr = 0.8716517857142857
INFO:tensorflow:Step: 500000 - Test batch complete: accr = 0.875202922077922
INFO:tensorflow:Step: 510000 - Test batch complete: accr = 0.8754058441558441
INFO:tensorflow:Step: 520000 - Test batch complete: accr = 0.8770292207792207
INFO:tensorflow:Step: 530000 - Test batch complete: accr = 0.8743912337662337
INFO:tensorflow:Step: 540000 - Test batch complete: accr = 0.8769277597402597
INFO:tensorflow:Step: 550000 - Test batch complete: accr = 0.8726663961038961
INFO:tensorflow:Step: 560000 - Test batch complete: accr = 0.877435064935065
INFO:tensorflow:Step: 570000 - Test batch complete: accr = 0.8665787337662337
INFO:tensorflow:Step: 580000 - Test batch complete: accr = 0.8791599025974026
INFO:tensorflow:Step: 590000 - Test batch complete: accr = 0.8797686688311688
INFO:tensorflow:Step: 600000 - Test batch complete: accr = 0.880275974025974
INFO:tensorflow:Step: 610000 - Test batch complete: accr = 0.8797686688311688
INFO:tensorflow:Step: 620000 - Test batch complete: accr = 0.8822037337662337
INFO:tensorflow:Step: 630000 - Test batch complete: accr = 0.8767248376623377
INFO:tensorflow:Step: 640000 - Test batch complete: accr = 0.8753043831168831
INFO:tensorflow:Step: 650000 - Test batch complete: accr = 0.8767248376623377
INFO:tensorflow:Step: 660000 - Test batch complete: accr = 0.8767248376623377
INFO:tensorflow:Step: 670000 - Test batch complete: accr = 0.8757102272727273
INFO:tensorflow:Step: 680000 - Test batch complete: accr = 0.8740868506493507
INFO:tensorflow:Step: 690000 - Test batch complete: accr = 0.8681006493506493
INFO:tensorflow:Step: 700000 - Test batch complete: accr = 0.875101461038961
INFO:tensorflow:Step: 710000 - Test batch complete: accr = 0.877435064935065
INFO:tensorflow:Step: 720000 - Test batch complete: accr = 0.8708400974025974
INFO:tensorflow:Step: 730000 - Test batch complete: accr = 0.8778409090909091
INFO:tensorflow:Step: 740000 - Test batch complete: accr = 0.8759131493506493
INFO:tensorflow:Step: 750000 - Test batch complete: accr = 0.8798701298701299
INFO:tensorflow:Step: 760000 - Test batch complete: accr = 0.8730722402597403
INFO:tensorflow:Step: 770000 - Test batch complete: accr = 0.8795657467532467
INFO:tensorflow:Step: 780000 - Test batch complete: accr = 0.8784496753246753
INFO:tensorflow:Step: 790000 - Test batch complete: accr = 0.8740868506493507
INFO:tensorflow:Step: 800000 - Test batch complete: accr = 0.8742897727272727
INFO:tensorflow:Step: 810000 - Test batch complete: accr = 0.8784496753246753
INFO:tensorflow:Step: 820000 - Test batch complete: accr = 0.874898538961039
INFO:tensorflow:Step: 830000 - Test batch complete: accr = 0.8741883116883117
INFO:tensorflow:Step: 840000 - Test batch complete: accr = 0.8770292207792207
INFO:tensorflow:Step: 850000 - Test batch complete: accr = 0.8786525974025974
INFO:tensorflow:Step: 860000 - Test batch complete: accr = 0.8793628246753247
INFO:tensorflow:Step: 870000 - Test batch complete: accr = 0.8778409090909091
INFO:tensorflow:Step: 880000 - Test batch complete: accr = 0.8712459415584416
INFO:tensorflow:Step: 890000 - Test batch complete: accr = 0.8727678571428571
INFO:tensorflow:Step: 900000 - Test batch complete: accr = 0.872362012987013
INFO:tensorflow:Step: 910000 - Test batch complete: accr = 0.8678977272727273
INFO:tensorflow:Step: 920000 - Test batch complete: accr = 0.8744926948051948
INFO:tensorflow:Step: 930000 - Test batch complete: accr = 0.8789569805194806
INFO:tensorflow:Step: 940000 - Test batch complete: accr = 0.8799715909090909
INFO:tensorflow:Step: 950000 - Test batch complete: accr = 0.8743912337662337
INFO:tensorflow:Step: 960000 - Test batch complete: accr = 0.8739853896103896
INFO:tensorflow:Step: 970000 - Test batch complete: accr = 0.8781452922077922
INFO:tensorflow:Step: 980000 - Test batch complete: accr = 0.8804788961038961
INFO:tensorflow:Step: 990000 - Test batch complete: accr = 0.8740868506493507
INFO:tensorflow:Step: 1000000 - Test batch complete: accr = 0.8782467532467533
INFO:tensorflow:Step: 1010000 - Test batch complete: accr = 0.8787540584415584
INFO:tensorflow:Step: 1020000 - Test batch complete: accr = 0.8787540584415584
INFO:tensorflow:Step: 1030000 - Test batch complete: accr = 0.875
INFO:tensorflow:Step: 1040000 - Test batch complete: accr = 0.8796672077922078
INFO:tensorflow:Step: 1050000 - Test batch complete: accr = 0.8736810064935064
INFO:tensorflow:Step: 1060000 - Test batch complete: accr = 0.8739853896103896
INFO:tensorflow:Step: 1070000 - Test batch complete: accr = 0.8782467532467533
INFO:tensorflow:Step: 1080000 - Test batch complete: accr = 0.8759131493506493
INFO:tensorflow:Step: 1090000 - Test batch complete: accr = 0.8791599025974026
INFO:tensorflow:Step: 1100000 - Test batch complete: accr = 0.8704342532467533
INFO:tensorflow:Step: 1110000 - Test batch complete: accr = 0.8796672077922078
INFO:tensorflow:Step: 1120000 - Test batch complete: accr = 0.8737824675324676
INFO:tensorflow:Step: 1130000 - Test batch complete: accr = 0.8693181818181818
INFO:tensorflow:Step: 1140000 - Test batch complete: accr = 0.8797686688311688
INFO:tensorflow:Step: 1150000 - Test batch complete: accr = 0.875
INFO:tensorflow:Step: 1160000 - Test batch complete: accr = 0.8736810064935064
INFO:tensorflow:Step: 1170000 - Test batch complete: accr = 0.8728693181818182
INFO:tensorflow:Step: 1180000 - Test batch complete: accr = 0.877637987012987
INFO:tensorflow:Step: 1190000 - Test batch complete: accr = 0.8762175324675324
INFO:tensorflow:Step: 1200000 - Test batch complete: accr = 0.8738839285714286
INFO:tensorflow:Step: 1210000 - Test batch complete: accr = 0.8738839285714286
INFO:tensorflow:Step: 1220000 - Test batch complete: accr = 0.8756087662337663
INFO:tensorflow:Step: 1230000 - Test batch complete: accr = 0.875101461038961
INFO:tensorflow:Step: 1240000 - Test batch complete: accr = 0.8804788961038961
INFO:tensorflow:Step: 1250000 - Test batch complete: accr = 0.8746956168831169
INFO:tensorflow:Step: 1260000 - Test batch complete: accr = 0.8726663961038961
INFO:tensorflow:Step: 1270000 - Test batch complete: accr = 0.869825487012987
INFO:tensorflow:Step: 1280000 - Test batch complete: accr = 0.875202922077922
INFO:tensorflow:Step: 1290000 - Test batch complete: accr = 0.8773336038961039
INFO:tensorflow:Step: 1300000 - Test batch complete: accr = 0.8733766233766234
INFO:tensorflow:Step: 1310000 - Test batch complete: accr = 0.8732751623376623
INFO:tensorflow:Step: 1320000 - Test batch complete: accr = 0.8741883116883117
INFO:tensorflow:Step: 1330000 - Test batch complete: accr = 0.8789569805194806
INFO:tensorflow:Step: 1340000 - Test batch complete: accr = 0.880073051948052
INFO:tensorflow:Step: 1350000 - Test batch complete: accr = 0.875
INFO:tensorflow:Step: 1360000 - Test batch complete: accr = 0.8754058441558441
INFO:tensorflow:Step: 1370000 - Test batch complete: accr = 0.8753043831168831
INFO:tensorflow:Step: 1380000 - Test batch complete: accr = 0.874797077922078
INFO:tensorflow:Step: 1390000 - Test batch complete: accr = 0.8721590909090909
INFO:tensorflow:Step: 1400000 - Test batch complete: accr = 0.8728693181818182
INFO:tensorflow:Step: 1410000 - Test batch complete: accr = 0.8784496753246753
INFO:tensorflow:Step: 1420000 - Test batch complete: accr = 0.874797077922078
INFO:tensorflow:Step: 1430000 - Test batch complete: accr = 0.8741883116883117
INFO:tensorflow:Step: 1440000 - Test batch complete: accr = 0.8755073051948052
INFO:tensorflow:Step: 1450000 - Test batch complete: accr = 0.8767248376623377
INFO:tensorflow:Step: 1460000 - Test batch complete: accr = 0.8766233766233766
INFO:tensorflow:Step: 1470000 - Test batch complete: accr = 0.8753043831168831
INFO:tensorflow:Step: 1480000 - Test batch complete: accr = 0.8754058441558441
INFO:tensorflow:Step: 1490000 - Test batch complete: accr = 0.880073051948052
INFO:tensorflow:Step: 1500000 - Test batch complete: accr = 0.8792613636363636
INFO:tensorflow:Step: 1510000 - Test batch complete: accr = 0.8721590909090909
INFO:tensorflow:Step: 1520000 - Test batch complete: accr = 0.8737824675324676
INFO:tensorflow:Step: 1530000 - Test batch complete: accr = 0.872564935064935
INFO:tensorflow:Step: 1540000 - Test batch complete: accr = 0.8758116883116883
INFO:tensorflow:Step: 1550000 - Test batch complete: accr = 0.8758116883116883
INFO:tensorflow:Step: 1560000 - Test batch complete: accr = 0.8780438311688312
INFO:tensorflow:Step: 1570000 - Test batch complete: accr = 0.8731737012987013
INFO:tensorflow:Step: 1580000 - Test batch complete: accr = 0.875
INFO:tensorflow:Step: 1590000 - Test batch complete: accr = 0.8770292207792207
INFO:tensorflow:Step: 1600000 - Test batch complete: accr = 0.877739448051948
INFO:tensorflow:Step: 1610000 - Test batch complete: accr = 0.8728693181818182
INFO:tensorflow:Step: 1620000 - Test batch complete: accr = 0.8742897727272727
INFO:tensorflow:Step: 1630000 - Test batch complete: accr = 0.8737824675324676
INFO:tensorflow:Step: 1640000 - Test batch complete: accr = 0.872362012987013
INFO:tensorflow:Step: 1650000 - Test batch complete: accr = 0.8781452922077922
INFO:tensorflow:Step: 1660000 - Test batch complete: accr = 0.8756087662337663
INFO:tensorflow:Step: 1670000 - Test batch complete: accr = 0.8754058441558441
INFO:tensorflow:Step: 1680000 - Test batch complete: accr = 0.8772321428571429
INFO:tensorflow:Step: 1690000 - Test batch complete: accr = 0.8768262987012987
INFO:tensorflow:Step: 1700000 - Test batch complete: accr = 0.8719561688311688
INFO:tensorflow:Step: 1710000 - Test batch complete: accr = 0.872463474025974
INFO:tensorflow:Step: 1720000 - Test batch complete: accr = 0.874898538961039
INFO:tensorflow:Step: 1730000 - Test batch complete: accr = 0.8684050324675324
INFO:tensorflow:Step: 1740000 - Test batch complete: accr = 0.8764204545454546
INFO:tensorflow:Step: 1750000 - Test batch complete: accr = 0.8791599025974026
INFO:tensorflow:Step: 1760000 - Test batch complete: accr = 0.8726663961038961
INFO:tensorflow:Step: 1770000 - Test batch complete: accr = 0.8756087662337663
INFO:tensorflow:Step: 1780000 - Test batch complete: accr = 0.8745941558441559
INFO:tensorflow:Step: 1790000 - Test batch complete: accr = 0.8701298701298701
INFO:tensorflow:Step: 1800000 - Test batch complete: accr = 0.8758116883116883
INFO:tensorflow:Step: 1810000 - Test batch complete: accr = 0.8746956168831169
INFO:tensorflow:Step: 1820000 - Test batch complete: accr = 0.8702313311688312
INFO:tensorflow:Step: 1830000 - Test batch complete: accr = 0.8728693181818182
INFO:tensorflow:Step: 1840000 - Test batch complete: accr = 0.8787540584415584
INFO:tensorflow:Step: 1850000 - Test batch complete: accr = 0.8716517857142857
INFO:tensorflow:Step: 1860000 - Test batch complete: accr = 0.880073051948052
INFO:tensorflow:Step: 1870000 - Test batch complete: accr = 0.877739448051948
INFO:tensorflow:Step: 1880000 - Test batch complete: accr = 0.8728693181818182
INFO:tensorflow:Step: 1890000 - Test batch complete: accr = 0.8773336038961039
INFO:tensorflow:Step: 1900000 - Test batch complete: accr = 0.880275974025974
INFO:tensorflow:Step: 1910000 - Test batch complete: accr = 0.8743912337662337
INFO:tensorflow:Step: 1920000 - Test batch complete: accr = 0.8781452922077922
INFO:tensorflow:Step: 1930000 - Test batch complete: accr = 0.8746956168831169
INFO:tensorflow:Step: 1940000 - Test batch complete: accr = 0.872564935064935
INFO:tensorflow:Step: 1950000 - Test batch complete: accr = 0.8718547077922078
INFO:tensorflow:Step: 1960000 - Test batch complete: accr = 0.8717532467532467
INFO:tensorflow:Step: 1970000 - Test batch complete: accr = 0.8716517857142857
INFO:tensorflow:Step: 1980000 - Test batch complete: accr = 0.8762175324675324
INFO:tensorflow:Step: 1990000 - Test batch complete: accr = 0.8728693181818182
INFO:tensorflow:Step: 2000000 - Test batch complete: accr = 0.8761160714285714
INFO:tensorflow:Step: 2010000 - Test batch complete: accr = 0.8769277597402597
INFO:tensorflow:Step: 2020000 - Test batch complete: accr = 0.8770292207792207
INFO:tensorflow:Step: 2030000 - Test batch complete: accr = 0.8770292207792207
INFO:tensorflow:Step: 2040000 - Test batch complete: accr = 0.8773336038961039
INFO:tensorflow:Step: 2050000 - Test batch complete: accr = 0.877637987012987
INFO:tensorflow:Step: 2060000 - Test batch complete: accr = 0.8719561688311688
INFO:tensorflow:Step: 2070000 - Test batch complete: accr = 0.8832183441558441
INFO:tensorflow:Step: 2080000 - Test batch complete: accr = 0.8780438311688312
INFO:tensorflow:Step: 2090000 - Test batch complete: accr = 0.877435064935065
INFO:tensorflow:Step: 2100000 - Test batch complete: accr = 0.8754058441558441
INFO:tensorflow:Step: 2110000 - Test batch complete: accr = 0.8735795454545454
INFO:tensorflow:Step: 2120000 - Test batch complete: accr = 0.8717532467532467
INFO:tensorflow:Step: 2130000 - Test batch complete: accr = 0.8707386363636364
INFO:tensorflow:Step: 2140000 - Test batch complete: accr = 0.8689123376623377
INFO:tensorflow:Step: 2150000 - Test batch complete: accr = 0.8713474025974026
INFO:tensorflow:Step: 2160000 - Test batch complete: accr = 0.8741883116883117
INFO:tensorflow:Step: 2170000 - Test batch complete: accr = 0.8785511363636364
INFO:tensorflow:Step: 2180000 - Test batch complete: accr = 0.8726663961038961
INFO:tensorflow:Step: 2190000 - Test batch complete: accr = 0.8731737012987013
INFO:tensorflow:Step: 2200000 - Test batch complete: accr = 0.8741883116883117
INFO:tensorflow:Step: 2210000 - Test batch complete: accr = 0.8761160714285714
INFO:tensorflow:Step: 2220000 - Test batch complete: accr = 0.872362012987013
INFO:tensorflow:Step: 2230000 - Test batch complete: accr = 0.8758116883116883
INFO:tensorflow:Step: 2240000 - Test batch complete: accr = 0.8733766233766234
INFO:tensorflow:Step: 2250000 - Test batch complete: accr = 0.8771306818181818
INFO:tensorflow:Step: 2260000 - Test batch complete: accr = 0.8762175324675324
INFO:tensorflow:Step: 2270000 - Test batch complete: accr = 0.8740868506493507
INFO:tensorflow:Step: 2280000 - Test batch complete: accr = 0.8707386363636364
INFO:tensorflow:Step: 2290000 - Test batch complete: accr = 0.8772321428571429
INFO:tensorflow:Step: 2300000 - Test batch complete: accr = 0.880073051948052
INFO:tensorflow:Epoch: accr = 0.8712459415584416
INFO:tensorflow:Saving model ... 
****************************************
Epochs: 1, each: 2302560 steps
Steps: 2302560
teacher_name: 	None
teacher_dset: 	False
dataset: 	/opt/datasets/deepsig/modulation_classification_resnet_train.rcrd
val_dataset: 	/opt/datasets/deepsig/modulation_classification_test_snr_30.rcrd
steps: 	2302560
epochs: 	1
test: 	False
no_mean: 	False
test_output: 	None
test_batches: 	154
batch_size: 	64
lr: 	0.01
model: 	resnet_twn
norespath: 	True
fconv_ch: 	32
nu_fconv: 	None
act_prec_fconv: 	None
k_1: 	3
lyr_conv: 	4
conv_ch: 	16
nu_conv: 	None
act_prec_conv: 	None
k_n: 	3
n_resblock: 	1
reslen: 	1
lyr_fc: 	3
fc_ch: 	128
nu_dense: 	None
act_prec_fc: 	None
twn_incr_act: 	None
no_filt_vgg: 	None
no_filts: 	None
gpus: 	1
directory: ../models/resnet_twn_FConv_32Co_FW_ActF_K1st3_Convs4_16Co_FW_ActF_Kn3_NResB1_ResLen1_FCs3_128Co_FW_ActF_norespath
****************************************
nu: [None, None, None, None, None, None, None, None], 
act_prec: [None, None, None, None, None, None, None, None], 
kernel: [3, 3, 3, 3, 3, 128, 128, 24]
False
Sun  5 Jan 17:23:25 AEDT 2020
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
WARNING:tensorflow:From run_cnn.py:82: DatasetV1.make_initializable_iterator (from tensorflow.python.data.ops.dataset_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `for ... in dataset:` to iterate over a dataset. If using `tf.estimator`, return the `Dataset` object directly from your input function. As a last resort, you can use `tf.compat.v1.data.make_initializable_iterator(dataset)`.
WARNING:tensorflow:From run_cnn.py:295: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:32: conv1d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.keras.layers.Conv1D` instead.
WARNING:tensorflow:From /home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/ops/init_ops.py:1251: calling VarianceScaling.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f5793150550>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f5793150550>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:42: batch_normalization (from tensorflow.python.layers.normalization) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.BatchNormalization instead.  In particular, `tf.control_dependencies(tf.GraphKeys.UPDATE_OPS)` should not be used (consult the `tf.keras.layers.batch_normalization` documentation).
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f5793150550>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f5793150550>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c2da9f98>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c2da9f98>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2e407b8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2e407b8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:76: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57fa001470>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57fa001470>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57fa001470>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57fa001470>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57fa001470>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57fa001470>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57fa001470>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57fa001470>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2d41400>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2d41400>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:79: max_pooling1d (from tensorflow.python.layers.pooling) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.MaxPooling1D instead.
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7f57c2d9f898>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7f57c2d9f898>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c2cfcdd8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c2cfcdd8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2cfcdd8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2cfcdd8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0a0bb00>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0a0bb00>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0a0bb00>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0a0bb00>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0a57978>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0a57978>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0a57978>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0a57978>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2cfcdd8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2cfcdd8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7f57c2da9940>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7f57c2da9940>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0b54780>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0b54780>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0b54780>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0b54780>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0a0bac8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0a0bac8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0a0bac8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0a0bac8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c08e5e48>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c08e5e48>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2d5d0f0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2d5d0f0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2bb5d30>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2bb5d30>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7f57c2d9f940>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7f57c2d9f940>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0894ba8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0894ba8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0894ba8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0894ba8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c2bb5d30>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c2bb5d30>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2bb5d30>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2bb5d30>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0a0bac8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7f57c0a0bac8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0a0bac8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c0a0bac8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2cbeeb8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2cbeeb8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7f57c2da9cc0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7f57c2da9cc0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:97: flatten (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.flatten instead.
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7f57c2e10ac8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7f57c2e10ac8>>: AttributeError: module 'gast' has no attribute 'Num'
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:8: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dense instead.
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f57c2d9fbe0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f57c2d9fbe0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2c52860>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2c52860>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f57c2da9cc0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f57c2da9cc0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2da9898>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7f57c2da9898>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f57c03e3908>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7f57c03e3908>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-01-05 17:23:32.318854: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcuda.so.1
2020-01-05 17:23:32.327705: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties: 
name: Tesla M40 major: 5 minor: 2 memoryClockRate(GHz): 1.112
pciBusID: 0000:82:00.0
2020-01-05 17:23:32.327952: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-05 17:23:32.329820: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-05 17:23:32.331171: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-05 17:23:32.331513: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-05 17:23:32.333510: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-05 17:23:32.334896: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-05 17:23:32.338448: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-05 17:23:32.340417: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0
2020-01-05 17:23:32.340883: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-01-05 17:23:32.429596: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x6ab01a0 executing computations on platform CUDA. Devices:
2020-01-05 17:23:32.429630: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Tesla M40, Compute Capability 5.2
2020-01-05 17:23:32.450482: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2200040000 Hz
2020-01-05 17:23:32.453855: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x6bc8ad0 executing computations on platform Host. Devices:
2020-01-05 17:23:32.453890: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
2020-01-05 17:23:32.456128: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties: 
name: Tesla M40 major: 5 minor: 2 memoryClockRate(GHz): 1.112
pciBusID: 0000:82:00.0
2020-01-05 17:23:32.456203: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-05 17:23:32.456227: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-05 17:23:32.456250: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-05 17:23:32.456274: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-05 17:23:32.456294: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-05 17:23:32.456315: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-05 17:23:32.456337: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-05 17:23:32.460363: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0
2020-01-05 17:23:32.460421: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-05 17:23:32.463269: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1181] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-01-05 17:23:32.463296: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1187]      0 
2020-01-05 17:23:32.463308: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 0:   N 
2020-01-05 17:23:32.466942: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10265 MB memory) -> physical GPU (device: 0, name: Tesla M40, pci bus id: 0000:82:00.0, compute capability: 5.2)
2020-01-05 17:23:32.786246: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
WARNING:tensorflow:From run_cnn.py:370: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Starting train loop at step 0
2020-01-05 17:23:37.645202: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-05 17:23:41.047376: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
INFO:tensorflow:Step: 10000 - Test batch complete: accr = 0.5770089285714286
INFO:tensorflow:Step: 20000 - Test batch complete: accr = 0.6799918831168831
INFO:tensorflow:Step: 30000 - Test batch complete: accr = 0.6699472402597403
INFO:tensorflow:Step: 40000 - Test batch complete: accr = 0.7606534090909091
INFO:tensorflow:Step: 50000 - Test batch complete: accr = 0.7351866883116883
INFO:tensorflow:Step: 60000 - Test batch complete: accr = 0.7621753246753247
INFO:tensorflow:Step: 70000 - Test batch complete: accr = 0.8049918831168831
INFO:tensorflow:Step: 80000 - Test batch complete: accr = 0.8157467532467533
INFO:tensorflow:Step: 90000 - Test batch complete: accr = 0.8003246753246753
INFO:tensorflow:Step: 100000 - Test batch complete: accr = 0.7954545454545454
INFO:tensorflow:Step: 110000 - Test batch complete: accr = 0.8403003246753247
INFO:tensorflow:Step: 120000 - Test batch complete: accr = 0.8319805194805194
INFO:tensorflow:Step: 130000 - Test batch complete: accr = 0.8375608766233766
INFO:tensorflow:Step: 140000 - Test batch complete: accr = 0.8442573051948052
INFO:tensorflow:Step: 150000 - Test batch complete: accr = 0.8345170454545454
INFO:tensorflow:Step: 160000 - Test batch complete: accr = 0.8571428571428571
INFO:tensorflow:Step: 170000 - Test batch complete: accr = 0.8451704545454546
INFO:tensorflow:Step: 180000 - Test batch complete: accr = 0.851359577922078
INFO:tensorflow:Step: 190000 - Test batch complete: accr = 0.8526785714285714
INFO:tensorflow:Step: 200000 - Test batch complete: accr = 0.8564326298701299
INFO:tensorflow:Step: 210000 - Test batch complete: accr = 0.854200487012987
INFO:tensorflow:Step: 220000 - Test batch complete: accr = 0.8506493506493507
INFO:tensorflow:Step: 230000 - Test batch complete: accr = 0.856635551948052
INFO:tensorflow:Step: 240000 - Test batch complete: accr = 0.8561282467532467
INFO:tensorflow:Step: 250000 - Test batch complete: accr = 0.8584618506493507
INFO:tensorflow:Step: 260000 - Test batch complete: accr = 0.856737012987013
INFO:tensorflow:Step: 270000 - Test batch complete: accr = 0.8659699675324676
INFO:tensorflow:Step: 280000 - Test batch complete: accr = 0.8580560064935064
INFO:tensorflow:Step: 290000 - Test batch complete: accr = 0.8637378246753247
INFO:tensorflow:Step: 300000 - Test batch complete: accr = 0.8538961038961039
INFO:tensorflow:Step: 310000 - Test batch complete: accr = 0.8643465909090909
INFO:tensorflow:Step: 320000 - Test batch complete: accr = 0.8607954545454546
INFO:tensorflow:Step: 330000 - Test batch complete: accr = 0.8587662337662337
INFO:tensorflow:Step: 340000 - Test batch complete: accr = 0.8617086038961039
INFO:tensorflow:Step: 350000 - Test batch complete: accr = 0.8625202922077922
INFO:tensorflow:Step: 360000 - Test batch complete: accr = 0.8571428571428571
INFO:tensorflow:Step: 370000 - Test batch complete: accr = 0.8590706168831169
INFO:tensorflow:Step: 380000 - Test batch complete: accr = 0.8604910714285714
INFO:tensorflow:Step: 390000 - Test batch complete: accr = 0.8655641233766234
INFO:tensorflow:Step: 400000 - Test batch complete: accr = 0.8622159090909091
INFO:tensorflow:Step: 410000 - Test batch complete: accr = 0.8515625
INFO:tensorflow:Step: 420000 - Test batch complete: accr = 0.8506493506493507
INFO:tensorflow:Step: 430000 - Test batch complete: accr = 0.8599837662337663
INFO:tensorflow:Step: 440000 - Test batch complete: accr = 0.8607954545454546
INFO:tensorflow:Step: 450000 - Test batch complete: accr = 0.8604910714285714
INFO:tensorflow:Step: 460000 - Test batch complete: accr = 0.8545048701298701
INFO:tensorflow:Step: 470000 - Test batch complete: accr = 0.8631290584415584
INFO:tensorflow:Step: 480000 - Test batch complete: accr = 0.8612012987012987
INFO:tensorflow:Step: 490000 - Test batch complete: accr = 0.8601866883116883
INFO:tensorflow:Step: 500000 - Test batch complete: accr = 0.8641436688311688
INFO:tensorflow:Step: 510000 - Test batch complete: accr = 0.8558238636363636
INFO:tensorflow:Step: 520000 - Test batch complete: accr = 0.8559253246753247
INFO:tensorflow:Step: 530000 - Test batch complete: accr = 0.8584618506493507
INFO:tensorflow:Step: 540000 - Test batch complete: accr = 0.859172077922078
INFO:tensorflow:Step: 550000 - Test batch complete: accr = 0.8550121753246753
INFO:tensorflow:Step: 560000 - Test batch complete: accr = 0.859476461038961
INFO:tensorflow:Step: 570000 - Test batch complete: accr = 0.861911525974026
INFO:tensorflow:Step: 580000 - Test batch complete: accr = 0.859172077922078
INFO:tensorflow:Step: 590000 - Test batch complete: accr = 0.859577922077922
INFO:tensorflow:Step: 600000 - Test batch complete: accr = 0.8630275974025974
INFO:tensorflow:Step: 610000 - Test batch complete: accr = 0.8579545454545454
INFO:tensorflow:Step: 620000 - Test batch complete: accr = 0.8587662337662337
INFO:tensorflow:Step: 630000 - Test batch complete: accr = 0.8604910714285714
INFO:tensorflow:Step: 640000 - Test batch complete: accr = 0.8580560064935064
INFO:tensorflow:Step: 650000 - Test batch complete: accr = 0.8577516233766234
INFO:tensorflow:Step: 660000 - Test batch complete: accr = 0.8555194805194806
INFO:tensorflow:Step: 670000 - Test batch complete: accr = 0.8533887987012987
INFO:tensorflow:Step: 680000 - Test batch complete: accr = 0.8626217532467533
INFO:tensorflow:Step: 690000 - Test batch complete: accr = 0.8610998376623377
INFO:tensorflow:Step: 700000 - Test batch complete: accr = 0.8587662337662337
INFO:tensorflow:Step: 710000 - Test batch complete: accr = 0.859577922077922
INFO:tensorflow:Step: 720000 - Test batch complete: accr = 0.8571428571428571
INFO:tensorflow:Step: 730000 - Test batch complete: accr = 0.8603896103896104
INFO:tensorflow:Step: 740000 - Test batch complete: accr = 0.8596793831168831
INFO:tensorflow:Step: 750000 - Test batch complete: accr = 0.8622159090909091
INFO:tensorflow:Step: 760000 - Test batch complete: accr = 0.8534902597402597
INFO:tensorflow:Step: 770000 - Test batch complete: accr = 0.861810064935065
INFO:tensorflow:Step: 780000 - Test batch complete: accr = 0.8616071428571429
INFO:tensorflow:Step: 790000 - Test batch complete: accr = 0.8639407467532467
INFO:tensorflow:Step: 800000 - Test batch complete: accr = 0.8631290584415584
INFO:tensorflow:Step: 810000 - Test batch complete: accr = 0.8612012987012987
INFO:tensorflow:Step: 820000 - Test batch complete: accr = 0.8636363636363636
INFO:tensorflow:Step: 830000 - Test batch complete: accr = 0.8578530844155844
INFO:tensorflow:Step: 840000 - Test batch complete: accr = 0.8596793831168831
INFO:tensorflow:Step: 850000 - Test batch complete: accr = 0.8623173701298701
INFO:tensorflow:Step: 860000 - Test batch complete: accr = 0.862012987012987
INFO:tensorflow:Step: 870000 - Test batch complete: accr = 0.856838474025974
INFO:tensorflow:Step: 880000 - Test batch complete: accr = 0.8579545454545454
INFO:tensorflow:Step: 890000 - Test batch complete: accr = 0.8585633116883117
INFO:tensorflow:Step: 900000 - Test batch complete: accr = 0.8596793831168831
INFO:tensorflow:Step: 910000 - Test batch complete: accr = 0.8587662337662337
INFO:tensorflow:Step: 920000 - Test batch complete: accr = 0.8607954545454546
INFO:tensorflow:Step: 930000 - Test batch complete: accr = 0.8684050324675324
INFO:tensorflow:Step: 940000 - Test batch complete: accr = 0.8579545454545454
INFO:tensorflow:Step: 950000 - Test batch complete: accr = 0.8572443181818182
INFO:tensorflow:Step: 960000 - Test batch complete: accr = 0.8585633116883117
INFO:tensorflow:Step: 970000 - Test batch complete: accr = 0.8605925324675324
INFO:tensorflow:Step: 980000 - Test batch complete: accr = 0.8622159090909091
INFO:tensorflow:Step: 990000 - Test batch complete: accr = 0.8606939935064936
INFO:tensorflow:Step: 1000000 - Test batch complete: accr = 0.8536931818181818
INFO:tensorflow:Step: 1010000 - Test batch complete: accr = 0.8587662337662337
INFO:tensorflow:Step: 1020000 - Test batch complete: accr = 0.8610998376623377
INFO:tensorflow:Step: 1030000 - Test batch complete: accr = 0.8603896103896104
INFO:tensorflow:Step: 1040000 - Test batch complete: accr = 0.8608969155844156
INFO:tensorflow:Step: 1050000 - Test batch complete: accr = 0.859375
INFO:tensorflow:Step: 1060000 - Test batch complete: accr = 0.8641436688311688
INFO:tensorflow:Step: 1070000 - Test batch complete: accr = 0.8580560064935064
INFO:tensorflow:Step: 1080000 - Test batch complete: accr = 0.8624188311688312
INFO:tensorflow:Step: 1090000 - Test batch complete: accr = 0.8610998376623377
INFO:tensorflow:Step: 1100000 - Test batch complete: accr = 0.862012987012987
INFO:tensorflow:Step: 1110000 - Test batch complete: accr = 0.8570413961038961
INFO:tensorflow:Step: 1120000 - Test batch complete: accr = 0.8607954545454546
INFO:tensorflow:Step: 1130000 - Test batch complete: accr = 0.8656655844155844
INFO:tensorflow:Step: 1140000 - Test batch complete: accr = 0.8583603896103896
INFO:tensorflow:Step: 1150000 - Test batch complete: accr = 0.8609983766233766
INFO:tensorflow:Step: 1160000 - Test batch complete: accr = 0.8632305194805194
INFO:tensorflow:Step: 1170000 - Test batch complete: accr = 0.8572443181818182
INFO:tensorflow:Step: 1180000 - Test batch complete: accr = 0.861911525974026
INFO:tensorflow:Step: 1190000 - Test batch complete: accr = 0.8598823051948052
INFO:tensorflow:Step: 1200000 - Test batch complete: accr = 0.862114448051948
INFO:tensorflow:Step: 1210000 - Test batch complete: accr = 0.8614042207792207
INFO:tensorflow:Step: 1220000 - Test batch complete: accr = 0.859273538961039
INFO:tensorflow:Step: 1230000 - Test batch complete: accr = 0.8679991883116883
INFO:tensorflow:Step: 1240000 - Test batch complete: accr = 0.8605925324675324
INFO:tensorflow:Step: 1250000 - Test batch complete: accr = 0.8617086038961039
INFO:tensorflow:Step: 1260000 - Test batch complete: accr = 0.8624188311688312
INFO:tensorflow:Step: 1270000 - Test batch complete: accr = 0.859577922077922
INFO:tensorflow:Step: 1280000 - Test batch complete: accr = 0.8642451298701299
INFO:tensorflow:Step: 1290000 - Test batch complete: accr = 0.8678977272727273
INFO:tensorflow:Step: 1300000 - Test batch complete: accr = 0.8597808441558441
INFO:tensorflow:Step: 1310000 - Test batch complete: accr = 0.8637378246753247
INFO:tensorflow:Step: 1320000 - Test batch complete: accr = 0.859172077922078
INFO:tensorflow:Step: 1330000 - Test batch complete: accr = 0.864752435064935
INFO:tensorflow:Step: 1340000 - Test batch complete: accr = 0.8640422077922078
INFO:tensorflow:Step: 1350000 - Test batch complete: accr = 0.8643465909090909
INFO:tensorflow:Step: 1360000 - Test batch complete: accr = 0.8602881493506493
INFO:tensorflow:Step: 1370000 - Test batch complete: accr = 0.8612012987012987
INFO:tensorflow:Step: 1380000 - Test batch complete: accr = 0.8596793831168831
INFO:tensorflow:Step: 1390000 - Test batch complete: accr = 0.8605925324675324
INFO:tensorflow:Step: 1400000 - Test batch complete: accr = 0.8614042207792207
INFO:tensorflow:Step: 1410000 - Test batch complete: accr = 0.8612012987012987
INFO:tensorflow:Step: 1420000 - Test batch complete: accr = 0.8635349025974026
INFO:tensorflow:Step: 1430000 - Test batch complete: accr = 0.8616071428571429
INFO:tensorflow:Step: 1440000 - Test batch complete: accr = 0.8624188311688312
INFO:tensorflow:Step: 1450000 - Test batch complete: accr = 0.8588676948051948
INFO:tensorflow:Step: 1460000 - Test batch complete: accr = 0.8609983766233766
INFO:tensorflow:Step: 1470000 - Test batch complete: accr = 0.8616071428571429
INFO:tensorflow:Step: 1480000 - Test batch complete: accr = 0.8636363636363636
INFO:tensorflow:Step: 1490000 - Test batch complete: accr = 0.8637378246753247
INFO:tensorflow:Step: 1500000 - Test batch complete: accr = 0.8616071428571429
INFO:tensorflow:Step: 1510000 - Test batch complete: accr = 0.8613027597402597
INFO:tensorflow:Step: 1520000 - Test batch complete: accr = 0.861911525974026
INFO:tensorflow:Step: 1530000 - Test batch complete: accr = 0.861810064935065
INFO:tensorflow:Step: 1540000 - Test batch complete: accr = 0.8634334415584416
INFO:tensorflow:Step: 1550000 - Test batch complete: accr = 0.8564326298701299
INFO:tensorflow:Step: 1560000 - Test batch complete: accr = 0.8587662337662337
INFO:tensorflow:Step: 1570000 - Test batch complete: accr = 0.8605925324675324
INFO:tensorflow:Step: 1580000 - Test batch complete: accr = 0.8664772727272727
INFO:tensorflow:Step: 1590000 - Test batch complete: accr = 0.8664772727272727
INFO:tensorflow:Step: 1600000 - Test batch complete: accr = 0.8654626623376623
INFO:tensorflow:Step: 1610000 - Test batch complete: accr = 0.8579545454545454
INFO:tensorflow:Step: 1620000 - Test batch complete: accr = 0.8575487012987013
INFO:tensorflow:Step: 1630000 - Test batch complete: accr = 0.8643465909090909
INFO:tensorflow:Step: 1640000 - Test batch complete: accr = 0.8665787337662337
INFO:tensorflow:Step: 1650000 - Test batch complete: accr = 0.8599837662337663
INFO:tensorflow:Step: 1660000 - Test batch complete: accr = 0.859476461038961
INFO:tensorflow:Step: 1670000 - Test batch complete: accr = 0.8623173701298701
INFO:tensorflow:Step: 1680000 - Test batch complete: accr = 0.8635349025974026
INFO:tensorflow:Step: 1690000 - Test batch complete: accr = 0.8628246753246753
INFO:tensorflow:Step: 1700000 - Test batch complete: accr = 0.8633319805194806
INFO:tensorflow:Step: 1710000 - Test batch complete: accr = 0.8572443181818182
INFO:tensorflow:Step: 1720000 - Test batch complete: accr = 0.859375
INFO:tensorflow:Step: 1730000 - Test batch complete: accr = 0.867390422077922
INFO:tensorflow:Step: 1740000 - Test batch complete: accr = 0.856939935064935
INFO:tensorflow:Step: 1750000 - Test batch complete: accr = 0.8605925324675324
INFO:tensorflow:Step: 1760000 - Test batch complete: accr = 0.8578530844155844
INFO:tensorflow:Step: 1770000 - Test batch complete: accr = 0.8552150974025974
INFO:tensorflow:Step: 1780000 - Test batch complete: accr = 0.8626217532467533
INFO:tensorflow:Step: 1790000 - Test batch complete: accr = 0.861911525974026
INFO:tensorflow:Step: 1800000 - Test batch complete: accr = 0.8659699675324676
INFO:tensorflow:Step: 1810000 - Test batch complete: accr = 0.8627232142857143
INFO:tensorflow:Step: 1820000 - Test batch complete: accr = 0.8561282467532467
INFO:tensorflow:Step: 1830000 - Test batch complete: accr = 0.8544034090909091
INFO:tensorflow:Step: 1840000 - Test batch complete: accr = 0.8617086038961039
INFO:tensorflow:Step: 1850000 - Test batch complete: accr = 0.856838474025974
INFO:tensorflow:Step: 1860000 - Test batch complete: accr = 0.8580560064935064
INFO:tensorflow:Step: 1870000 - Test batch complete: accr = 0.8660714285714286
INFO:tensorflow:Step: 1880000 - Test batch complete: accr = 0.8604910714285714
INFO:tensorflow:Step: 1890000 - Test batch complete: accr = 0.8583603896103896
INFO:tensorflow:Step: 1900000 - Test batch complete: accr = 0.8649553571428571
INFO:tensorflow:Step: 1910000 - Test batch complete: accr = 0.8629261363636364
INFO:tensorflow:Step: 1920000 - Test batch complete: accr = 0.8573457792207793
INFO:tensorflow:Step: 1930000 - Test batch complete: accr = 0.8651582792207793
INFO:tensorflow:Step: 1940000 - Test batch complete: accr = 0.8687094155844156
INFO:tensorflow:Step: 1950000 - Test batch complete: accr = 0.8599837662337663
INFO:tensorflow:Step: 1960000 - Test batch complete: accr = 0.862012987012987
INFO:tensorflow:Step: 1970000 - Test batch complete: accr = 0.8631290584415584
INFO:tensorflow:Step: 1980000 - Test batch complete: accr = 0.8579545454545454
INFO:tensorflow:Step: 1990000 - Test batch complete: accr = 0.8551136363636364
INFO:tensorflow:Step: 2000000 - Test batch complete: accr = 0.8602881493506493
INFO:tensorflow:Step: 2010000 - Test batch complete: accr = 0.8608969155844156
INFO:tensorflow:Step: 2020000 - Test batch complete: accr = 0.8552150974025974
INFO:tensorflow:Step: 2030000 - Test batch complete: accr = 0.8604910714285714
INFO:tensorflow:Step: 2040000 - Test batch complete: accr = 0.8634334415584416
INFO:tensorflow:Step: 2050000 - Test batch complete: accr = 0.859273538961039
INFO:tensorflow:Step: 2060000 - Test batch complete: accr = 0.8580560064935064
INFO:tensorflow:Step: 2070000 - Test batch complete: accr = 0.8590706168831169
INFO:tensorflow:Step: 2080000 - Test batch complete: accr = 0.8603896103896104
INFO:tensorflow:Step: 2090000 - Test batch complete: accr = 0.8572443181818182
INFO:tensorflow:Step: 2100000 - Test batch complete: accr = 0.8577516233766234
INFO:tensorflow:Step: 2110000 - Test batch complete: accr = 0.8638392857142857
INFO:tensorflow:Step: 2120000 - Test batch complete: accr = 0.8608969155844156
INFO:tensorflow:Step: 2130000 - Test batch complete: accr = 0.8633319805194806
INFO:tensorflow:Step: 2140000 - Test batch complete: accr = 0.864752435064935
INFO:tensorflow:Step: 2150000 - Test batch complete: accr = 0.8599837662337663
INFO:tensorflow:Step: 2160000 - Test batch complete: accr = 0.856737012987013
INFO:tensorflow:Step: 2170000 - Test batch complete: accr = 0.8573457792207793
INFO:tensorflow:Step: 2180000 - Test batch complete: accr = 0.8631290584415584
INFO:tensorflow:Step: 2190000 - Test batch complete: accr = 0.8586647727272727
INFO:tensorflow:Step: 2200000 - Test batch complete: accr = 0.8624188311688312
INFO:tensorflow:Step: 2210000 - Test batch complete: accr = 0.8572443181818182
INFO:tensorflow:Step: 2220000 - Test batch complete: accr = 0.8638392857142857
INFO:tensorflow:Step: 2230000 - Test batch complete: accr = 0.8608969155844156
INFO:tensorflow:Step: 2240000 - Test batch complete: accr = 0.859375
INFO:tensorflow:Step: 2250000 - Test batch complete: accr = 0.864448051948052
INFO:tensorflow:Step: 2260000 - Test batch complete: accr = 0.8587662337662337
INFO:tensorflow:Step: 2270000 - Test batch complete: accr = 0.8578530844155844
INFO:tensorflow:Step: 2280000 - Test batch complete: accr = 0.8553165584415584
INFO:tensorflow:Step: 2290000 - Test batch complete: accr = 0.854099025974026
INFO:tensorflow:Step: 2300000 - Test batch complete: accr = 0.8629261363636364
INFO:tensorflow:Epoch: accr = 0.8626217532467533
INFO:tensorflow:Saving model ... 
****************************************
Epochs: 1, each: 2302560 steps
Steps: 2302560
teacher_name: 	None
teacher_dset: 	False
dataset: 	/opt/datasets/deepsig/modulation_classification_resnet_train.rcrd
val_dataset: 	/opt/datasets/deepsig/modulation_classification_test_snr_30.rcrd
steps: 	2302560
epochs: 	1
test: 	False
no_mean: 	False
test_output: 	None
test_batches: 	154
batch_size: 	64
lr: 	0.01
model: 	resnet_twn
norespath: 	True
fconv_ch: 	32
nu_fconv: 	None
act_prec_fconv: 	None
k_1: 	3
lyr_conv: 	4
conv_ch: 	32
nu_conv: 	None
act_prec_conv: 	None
k_n: 	3
n_resblock: 	1
reslen: 	1
lyr_fc: 	3
fc_ch: 	128
nu_dense: 	None
act_prec_fc: 	None
twn_incr_act: 	None
no_filt_vgg: 	None
no_filts: 	None
gpus: 	1
directory: ../models/resnet_twn_FConv_32Co_FW_ActF_K1st3_Convs4_32Co_FW_ActF_Kn3_NResB1_ResLen1_FCs3_128Co_FW_ActF_norespath
****************************************
nu: [None, None, None, None, None, None, None, None], 
act_prec: [None, None, None, None, None, None, None, None], 
kernel: [3, 3, 3, 3, 3, 128, 128, 24]
False
Mon  6 Jan 15:06:39 AEDT 2020
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:516: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:517: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:518: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:519: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:520: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/framework/dtypes.py:525: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:541: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint8 = np.dtype([("qint8", np.int8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:542: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint8 = np.dtype([("quint8", np.uint8, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:543: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint16 = np.dtype([("qint16", np.int16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:544: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_quint16 = np.dtype([("quint16", np.uint16, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:545: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  _np_qint32 = np.dtype([("qint32", np.int32, 1)])
/home/ramin/venv3_radio/lib/python3.6/site-packages/tensorboard/compat/tensorflow_stub/dtypes.py:550: FutureWarning: Passing (type, 1) or '1type' as a synonym of type is deprecated; in a future version of numpy, it will be understood as (type, (1,)) / '(1,)type'.
  np_resource = np.dtype([("resource", np.ubyte, 1)])
WARNING:tensorflow:From run_cnn.py:82: DatasetV1.make_initializable_iterator (from tensorflow.python.data.ops.dataset_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use `for ... in dataset:` to iterate over a dataset. If using `tf.estimator`, return the `Dataset` object directly from your input function. As a last resort, you can use `tf.compat.v1.data.make_initializable_iterator(dataset)`.
WARNING:tensorflow:From run_cnn.py:295: add_dispatch_support.<locals>.wrapper (from tensorflow.python.ops.array_ops) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.where in 2.0, which has the same broadcast rule as np.where
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:32: conv1d (from tensorflow.python.layers.convolutional) is deprecated and will be removed in a future version.
Instructions for updating:
Use `tf.keras.layers.Conv1D` instead.
WARNING:tensorflow:From /home/ramin/venv3_radio/lib/python3.6/site-packages/tensorflow/python/ops/init_ops.py:1251: calling VarianceScaling.__init__ (from tensorflow.python.ops.init_ops) with dtype is deprecated and will be removed in a future version.
Instructions for updating:
Call initializer instance with the dtype argument instead of passing it to the constructor
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4a6638518>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4a6638518>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:42: batch_normalization (from tensorflow.python.layers.normalization) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.BatchNormalization instead.  In particular, `tf.control_dependencies(tf.GraphKeys.UPDATE_OPS)` should not be used (consult the `tf.keras.layers.batch_normalization` documentation).
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4a6638518>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4a6638518>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d6251f98>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d6251f98>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d61db940>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d61db940>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:76: The name tf.variable_scope is deprecated. Please use tf.compat.v1.variable_scope instead.

WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d61e9e48>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d61e9e48>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d61e9e48>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d61e9e48>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d61e91d0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d61e91d0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d61e91d0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d61e91d0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd50d4e9470>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd50d4e9470>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:79: max_pooling1d (from tensorflow.python.layers.pooling) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.MaxPooling1D instead.
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fd4d6247898>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fd4d6247898>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d614ac88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d614ac88>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d614ac88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d614ac88>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d403e4a8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d403e4a8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d403e4a8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d403e4a8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3f68400>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3f68400>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3f68400>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3f68400>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd50d4e93c8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd50d4e93c8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fd4d6251940>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fd4d6251940>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d614ac88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d614ac88>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d614ac88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d614ac88>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3c70eb8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3c70eb8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3e119e8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3e119e8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3e7ee80>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3e7ee80>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3c9c668>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3c9c668>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3f97f28>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3f97f28>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fd4d6247908>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fd4d6247908>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3c399b0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3c399b0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3c399b0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3c399b0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d614ac88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d614ac88>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d614ac88>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d614ac88>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3e114e0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Conv1D.call of <tensorflow.python.layers.convolutional.Conv1D object at 0x7fd4d3e114e0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3e114e0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3e114e0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d403e4a8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d403e4a8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fd4d6251cc0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Pooling1D.call of <tensorflow.python.layers.pooling.MaxPooling1D object at 0x7fd4d6251cc0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:97: flatten (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.flatten instead.
WARNING:tensorflow:Entity <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7fd4d62b8ac8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Flatten.call of <tensorflow.python.layers.core.Flatten object at 0x7fd4d62b8ac8>>: AttributeError: module 'gast' has no attribute 'Num'
WARNING:tensorflow:From /home/ramin/radio_modulation/train_tnn/resnet.py:8: dense (from tensorflow.python.layers.core) is deprecated and will be removed in a future version.
Instructions for updating:
Use keras.layers.dense instead.
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fd4d6247ba8>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fd4d6247ba8>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3c399b0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d3c399b0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fd4d6251cc0>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fd4d6251cc0>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d6251898>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method BatchNormalization.call of <tensorflow.python.layers.normalization.BatchNormalization object at 0x7fd4d6251898>>: AssertionError: Bad argument number for Name: 3, expecting 4
WARNING:tensorflow:Entity <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fd4d3ecaa90>> could not be transformed and will be executed as-is. Please report this to the AutgoGraph team. When filing the bug, set the verbosity to 10 (on Linux, `export AUTOGRAPH_VERBOSITY=10`) and attach the full output. Cause: converting <bound method Dense.call of <tensorflow.python.layers.core.Dense object at 0x7fd4d3ecaa90>>: AssertionError: Bad argument number for Name: 3, expecting 4
2020-01-06 15:06:45.828035: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcuda.so.1
2020-01-06 15:06:45.839544: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties: 
name: Tesla M40 major: 5 minor: 2 memoryClockRate(GHz): 1.112
pciBusID: 0000:82:00.0
2020-01-06 15:06:45.839824: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-06 15:06:45.841677: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-06 15:06:45.843015: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-06 15:06:45.843412: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-06 15:06:45.845487: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-06 15:06:45.846816: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-06 15:06:45.850276: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-06 15:06:45.853029: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0
2020-01-06 15:06:45.853428: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-01-06 15:06:45.953687: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x6c7aa30 executing computations on platform CUDA. Devices:
2020-01-06 15:06:45.953741: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): Tesla M40, Compute Capability 5.2
2020-01-06 15:06:45.978335: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2200040000 Hz
2020-01-06 15:06:45.982021: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x6d935d0 executing computations on platform Host. Devices:
2020-01-06 15:06:45.982055: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
2020-01-06 15:06:45.984573: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1640] Found device 0 with properties: 
name: Tesla M40 major: 5 minor: 2 memoryClockRate(GHz): 1.112
pciBusID: 0000:82:00.0
2020-01-06 15:06:45.984647: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-06 15:06:45.984674: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-06 15:06:45.984695: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcufft.so.10.0
2020-01-06 15:06:45.984719: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcurand.so.10.0
2020-01-06 15:06:45.984742: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusolver.so.10.0
2020-01-06 15:06:45.984764: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcusparse.so.10.0
2020-01-06 15:06:45.984787: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
2020-01-06 15:06:45.988987: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1763] Adding visible gpu devices: 0
2020-01-06 15:06:45.989051: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudart.so.10.0
2020-01-06 15:06:45.991980: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1181] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-01-06 15:06:45.992007: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1187]      0 
2020-01-06 15:06:45.992019: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1200] 0:   N 
2020-01-06 15:06:45.995423: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1326] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 10772 MB memory) -> physical GPU (device: 0, name: Tesla M40, pci bus id: 0000:82:00.0, compute capability: 5.2)
2020-01-06 15:06:46.270257: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
WARNING:tensorflow:From run_cnn.py:370: checkpoint_exists (from tensorflow.python.training.checkpoint_management) is deprecated and will be removed in a future version.
Instructions for updating:
Use standard file APIs to check for files with this prefix.
INFO:tensorflow:Starting train loop at step 0
2020-01-06 15:06:51.192604: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcublas.so.10.0
2020-01-06 15:06:54.251514: I tensorflow/stream_executor/platform/default/dso_loader.cc:42] Successfully opened dynamic library libcudnn.so.7
INFO:tensorflow:Step: 10000 - Test batch complete: accr = 0.7167207792207793
INFO:tensorflow:Step: 20000 - Test batch complete: accr = 0.7478693181818182
INFO:tensorflow:Step: 30000 - Test batch complete: accr = 0.7100243506493507
INFO:tensorflow:Step: 40000 - Test batch complete: accr = 0.734375
INFO:tensorflow:Step: 50000 - Test batch complete: accr = 0.827922077922078
INFO:tensorflow:Step: 60000 - Test batch complete: accr = 0.8007305194805194
INFO:tensorflow:Step: 70000 - Test batch complete: accr = 0.8337053571428571
INFO:tensorflow:Step: 80000 - Test batch complete: accr = 0.8544034090909091
INFO:tensorflow:Step: 90000 - Test batch complete: accr = 0.8626217532467533
INFO:tensorflow:Step: 100000 - Test batch complete: accr = 0.8508522727272727
INFO:tensorflow:Step: 110000 - Test batch complete: accr = 0.8850446428571429
INFO:tensorflow:Step: 120000 - Test batch complete: accr = 0.8993506493506493
INFO:tensorflow:Step: 130000 - Test batch complete: accr = 0.908786525974026
INFO:tensorflow:Step: 140000 - Test batch complete: accr = 0.9111201298701299
INFO:tensorflow:Step: 150000 - Test batch complete: accr = 0.9133522727272727
INFO:tensorflow:Step: 160000 - Test batch complete: accr = 0.9201501623376623
INFO:tensorflow:Step: 170000 - Test batch complete: accr = 0.9134537337662337
INFO:tensorflow:Step: 180000 - Test batch complete: accr = 0.9303977272727273
INFO:tensorflow:Step: 190000 - Test batch complete: accr = 0.9213676948051948
INFO:tensorflow:Step: 200000 - Test batch complete: accr = 0.9224837662337663
INFO:tensorflow:Step: 210000 - Test batch complete: accr = 0.9264407467532467
INFO:tensorflow:Step: 220000 - Test batch complete: accr = 0.9276582792207793
INFO:tensorflow:Step: 230000 - Test batch complete: accr = 0.9256290584415584
INFO:tensorflow:Step: 240000 - Test batch complete: accr = 0.9261363636363636
INFO:tensorflow:Step: 250000 - Test batch complete: accr = 0.9301948051948052
INFO:tensorflow:Step: 260000 - Test batch complete: accr = 0.9285714285714286
INFO:tensorflow:Step: 270000 - Test batch complete: accr = 0.9289772727272727
INFO:tensorflow:Step: 280000 - Test batch complete: accr = 0.9293831168831169
INFO:tensorflow:Step: 290000 - Test batch complete: accr = 0.9343547077922078
INFO:tensorflow:Step: 300000 - Test batch complete: accr = 0.9300933441558441
INFO:tensorflow:Step: 310000 - Test batch complete: accr = 0.926948051948052
INFO:tensorflow:Step: 320000 - Test batch complete: accr = 0.9310064935064936
INFO:tensorflow:Step: 330000 - Test batch complete: accr = 0.9292816558441559
INFO:tensorflow:Step: 340000 - Test batch complete: accr = 0.934760551948052
INFO:tensorflow:Step: 350000 - Test batch complete: accr = 0.9329342532467533
INFO:tensorflow:Step: 360000 - Test batch complete: accr = 0.9333400974025974
INFO:tensorflow:Step: 370000 - Test batch complete: accr = 0.934760551948052
